File: SPA/src/app/_core/models/report/storageSumReportParam.ts
export interface StorageSumReportParam {
    date_kind: string;
    date_start: string;
    date_end: string;
    brandname: string;
    cusid: string;
    manno: string;
    rmodel: string;
    tolcls: string;
    purno: string;
    material: string;
    kind: string;
    etd_start: string;
    etd_end: string;
    size: string;
}

export interface StorageSumDetailReportParam {
    manno: string;
    purno: string;
    size: string;
}

File: API/DTOs/Report/StorageSumReportParam.cs


namespace API.DTOs.Report
{
    public class StorageSumReportParam
    {
        public string date_kind { get; set; }
        public DateTime? date_start { get; set; }
        public DateTime? date_end { get; set; }
        public string brandname { get; set; }
        public string cusid { get; set; }
        public string manno { get; set; }
        public string rmodel { get; set; }
        public string tolcls { get; set; }
        public string purno { get; set; }
        public string material { get; set; }
        public string kind { get; set; }
        public DateTime? etd_start { get; set; }
        public DateTime? etd_end { get; set; }
        public string size { get; set; }
    }

    public class StorageSumDetailReportParam
    {
        public string manno { get; set; }

        public string purno { get; set; }

        public string size { get; set; }
    }
}